### PR TITLE
About us 2 (image ang progress-bar)

### PR DESCRIPTION
**Pull request by EvgenySivaykin**
All done

### CODE REVIEW

**Review by zilva149 — CHANGES_REQUESTED** [review]
Looks good, but needs some minor changes. I will comment those places in a minute. Also don't forget to do git merge origin before doing final commit.

File: css/components/progress-bar.css [hunk 1]
```diff
@@ -2,6 +2,7 @@
     display: flex;
     flex-direction: column;
     gap: 10px;
+    padding-bottom: 1em;
 }
```
Comment (zilva149) [inline css/components/progress-bar.css:6]: progress-bar text `p` should be 14px.

**Review by zilva149 — CHANGES_REQUESTED** [review]

**Comment by EvgenySivaykin** [comment]
> 

OK.

File: css/pages/home.css [hunk 2]
```diff
@@ -143,6 +143,20 @@
...
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+}
+.about-us-2-img {
+    width: 100%;
+}
+.about-us-2-text {
+    padding-bottom: 1em;
+}
```
Comment (zilva149) [inline css/pages/home.css:158]: Pseudo elements don't work, because of padding. There should be no padding on this element.

**Review by zilva149 — CHANGES_REQUESTED** [review]

File: css/components/progress-bar.css (hunk 1 above)
Comment (EvgenySivaykin) [reply css/components/progress-bar.css:6]: > progress-bar text `p` should be 14px.

I check original - there text size - p => 18 px (always).

**Review by liepsneletech — APPROVED** [review]
Look nice!

**Review by Dorenus — APPROVED** [review]

**Review by zilva149 — APPROVED** [review]

**Pull request merged**